integration/helpers/create-fixture.tsx

```diff
@@ -307,6 +307,15 @@ export async function createAppFixture(fixture: Fixture) {
        */
       getHtml: (selector?: string) => getHtml(page, selector),
 
+      /**
+       * Get the attribute from an HTML element given a selector.
+       *
+       * @param selector CSS Selector for the element's HTML you want
+       * @param attribute Name of the attribute you want
+       */
+      getAttribute: async (selector: string, attribute: string) =>
```

Comment:
`app.getAttribute` is a confusing interface. Is it getting the attribute of the app? It'd be clearer if we had an interface like:

```
let html = app.getHTML();
let el = getElement(html, selector);
let attr = getAttribute(el, attrName);
```

Can we make this more clear?

Reply:
Updated, though I omitted `getAttribute` since `getElement` returns a cheerio instance we can use directly via `element.attr("#whatever")`